apps/root/src/pages/strategy-guardian-detail/vault-investment-data/components/financial-data/index.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { DisplayStrategy, EarnPosition } from 'common-types';
+import { FormattedMessage } from 'react-intl';
+import { ContainerBox, DividerBorder1, Typography } from 'ui-library';
+import ExpectedReturns, { StrategyReturnPeriods } from '../expected-returns';
+import FinancialOverview from '../financial-overview';
+
+interface FinancialDataProps {
+  userPositions: EarnPosition[];
+  strategy: DisplayStrategy;
+}
+
+const FinancialData = ({ userPositions, strategy }: FinancialDataProps) => {
+  return (
+    <>
+      <FinancialOverview userPositions={userPositions} strategy={strategy} />
+      <DividerBorder1 />
+      <ContainerBox flexDirection="column" gap={2}>
```

Comment:
This actually does not use a gap.

Reply:
I'm seeing 8px between the 'Expected returns' title and the periods 🤔

Reply:
Oh right, its the expected returns that dont use it